carma_wm/src/WMListener.cpp

```diff
@@ -35,6 +36,7 @@ WMListener::WMListener(bool multi_thread) : worker_(std::unique_ptr<WMListenerWo
   map_sub_ = nh_.subscribe("semantic_map", 2, &WMListenerWorker::mapCallback, worker_.get());
   route_sub_ = nh_.subscribe("route", 1, &WMListenerWorker::routeCallback, worker_.get());
   roadway_objects_sub_ = nh_.subscribe("roadway_objects", 1, &WMListenerWorker::roadwayObjectListCallback, worker_.get());
+  traffic_spat_sub_ = nh_.subscribe("incoming_spat", 1, &WMListenerWorker::incomingSpatCallback, worker_.get());
```

Comment:
Since the system might operate in the presence of multiple intersections in the future, you should use a larger queue here to account for the possibility of multiple SPaT sources. I think 10 would probably be sufficient as I doubt we will encounter more unique intersections than that at any point.